.github/workflows/zfs-tests-functional.yml

```diff
@@ -55,12 +58,9 @@ jobs:
             sudo udevadm control --reload-rules
           fi
         fi
-        # Workaround to provide additional free space for testing.
-        #   https://github.com/actions/virtual-environments/issues/2840
-        sudo rm -rf /usr/share/dotnet
-        sudo rm -rf /opt/ghc
-        sudo rm -rf "/usr/local/share/boost"
-        sudo rm -rf "$AGENT_TOOLSDIRECTORY"
+    - name: Report disk space
+      run: |
+        sudo df --human-readable --sync /
```

Comment:
Also, what's the point of syncing here? We've left V5 long behind us. (And why are you running it as root? This is better served as just `df -h /`)